### PR TITLE
Streaming timeline printer

### PR DESCRIPTION
**Pull request by darkrodry**
Demo to print the user timeline in real time and cherry pick of some refactors from the previous PR https://github.com/DanielaSfregola/twitter4s-demo/pull/18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ TwitterStreamingClient Examples
 --------------------------------
 - [UserMentionStream](https://github.com/DanielaSfregola/twitter4s-demo/blob/master/src/main/scala/streaming/UserMentionStream.scala) receive stream of tweets mentioning yourself.
 - [StreamingWithLogging](https://github.com/DanielaSfregola/twitter4s-demo/blob/master/src/main/scala/streaming/StreamingWithLogging.scala) example on how to use logging in your twitter4s application.
+- [TweetTimelineStreamPrinter](https://github.com/DanielaSfregola/twitter4s-demo/blob/master/src/main/scala/streaming/TweetTimelineStreamPrinter.scala) print the tweets in your timeline using stream.
```
Comment (DanielaSfregola) [inline README.md]: You should also add a description for SampleStatusesHashtagPrinter

File: src/main/scala/streaming/SampleStatusesHashtagPrinter.scala [hunk 2]
```diff
@@ -8,23 +8,19 @@ object SampleStatusesHashtagPrinter extends App {
 
   val client = TwitterStreamingClient()
```
Comment (DanielaSfregola) [inline src/main/scala/streaming/SampleStatusesHashtagPrinter.scala]: I would add the same TODO comment for consistency:
`// TODO - Make sure to define your consumer and access tokens!`

**Review by DanielaSfregola — CHANGES_REQUESTED** [review]
Hi @darkrodry,
thanks for your contribution!

Only minor comments and a description missing in the README.

Once these are fixed, we are good to go!

**Comment by darkrodry** [comment]
Changes are ready!

**Review by DanielaSfregola — APPROVED** [review]
Hi @darkrodry,
thanks for the changes and thanks for your contribution!

LGTM

**Pull request merged**